src/tools/dartpad/index.md

```diff
@@ -115,54 +120,8 @@ for (var char in 'hello'.split('')) {
 </ol>
 
 
-### Create a web app {#step-3-web}
```

Comment:
this could optionally be rewritten to include clicking the checkbox, unless we're intending to deemphasize that DartPad can do this.

Reply:
Without a very basic HTML app to start from, it's hard to tell them what to do. Plus this site doesn't cover web programming. So I opted to simplify. I could point them to webdev.dartlang.org, though...

Reply:
Without a very basic HTML app to start from, it's hard to tell them what to do. Plus this site doesn't cover web programming. So I opted to simplify. I could point them to webdev.dartlang.org, though...